test/test_cuda.py

```diff
@@ -3828,7 +3824,7 @@ def alloc(n):
             # expandable_segment blocks can be in the free list when this is called.
             alloc(80)
         finally:
-            orig = torch.cuda.get_per_process_memory_fraction(0)
+            torch.cuda.get_per_process_memory_fraction(0)
```

Comment:
What's the point of this? to compare to before? Is there a missing assert/

Reply:
Not sure, we can try to remove the lines.